### ISSUE TITLE
SQL Error

### ISSUE THREAD

**Issue by ghost** [MatissJanis/oc-mail#13]
tried to install this on a new installation

Update failed

"SQLSTATE[HY000]: General error: 1 near "MODIFY": syntax error (SQL: ALTER TABLE `mja_mail_email_log` MODIFY `code` VARCHAR(255) NULL;)" on line 624 of [...]\vendor\laravel\framework\src\Illuminate\Database\Connection.php

**Comment by MatissJanis** [comment]
Thanks! I will try to find time and work on this bug this weekend.

**Comment by MatissJanis** [comment]
What kind of database are You using? I just tried re-running this migration and everything seemed to work for me in my environment.

**Comment by ghost** [comment]
Sqlite3

**Comment by MatissJanis** [comment]
That may be exactly the problem we're experiencing. I think that Sqlite doesn't support this kind of queries. Would you mind taking a look at this and submitting a PR with fix for this? Something along the lines of .. should work:

```
if ($database === 'sqlite') {
    $query = '...';
} else {
    $query = 'ALTER TABLE mja_mail_email_log MODIFY code VARCHAR(255) NULL';
}
```

**Comment by nozreh** [comment]
If anyone encountered this error just comment out the ALTER statement in the updates files 'make_body_column_..  and 'make_code_column_..

**Comment by MatissJanis** [comment]
Fixed in v1.2.1
